frontend/src/components/dashboard/create/email/EmailTemplate.tsx

```diff
@@ -190,9 +192,7 @@ const EmailTemplate = ({
         </div>
 
         <div>
-          <h4 className={styles.replyToHeader}>
-            Replies <em>optional</em>
-          </h4>
+          <h4 className={styles.replyToHeader}>Replies</h4>
           <p>
             All replies will be directed to the email address indicated below
```

Comment:
Then we can change this to: `If left blank, replies will be directed to ${userEmail}`